content/docs/release-notes.md

```diff
@@ -2,7 +2,60 @@
 title: Release Notes
 ---
 
-## 2022-07-11
+## 2022-07-19 Storage release
+
+### What's new
+
+* Safekeeper: Back up WAL to S3 for disaster recovery.
+
+* Safekeeper: Download WAL from S3 on demand.
+
+* Propagate postgres authentication errors to the user.
+
+* Rename `contrib/zenith` to `contrib/neon`.
+* Rename custom GUCs:
+    * `zenith.page_server_connstring` -> `neon.pageserver_connstring`
+    * `zenith.zenith_tenant` -> `neon.tenant_id`
+    * `zenith.zenith_timeline` -> `neon.timeline_id`
+    * `zenith.max_cluster_size` -> `neon.max_cluster_size`
```

Comment:
Not sure, but IIRC this (and many other changes listed as the last release) was done quite a while ago as we updated compute version periodically. Not important, probably, but still

Reply:
Good point.
Should we write release notes for every compute version update in the future?